docs/developer-docs/daos/nns/neuron-management.md

```diff
@@ -0,0 +1,268 @@
+A neuron represents a participant of the NNS DAO and can participate in governance. Concretely, a neuron can submit proposals or vote on them if its dissolve delay is at least 6 months. 
+A neuron can either directly vote to adopt or reject a given proposal or it can _follow_ other neurons for certain decisions - which will automatically cast a ballot for the neuron if enough of the followees voted.
+
+**Vote directly**
+To vote directly, first find out what proposals can be voted on and then find the ID of the proposal you want to vote on.
+The following are the relevant NNS governance API and record that can also be found in the candid file.
+```
+service : (Governance) -> {
+  get_pending_proposals : () -> (vec ProposalInfo) query;
+};
+type ProposalInfo = record {
+  id : opt NeuronId;
+  ...
+};
+ ```
+First use the `get_pending_proposals` API to get the `ProposalInfo` of all proposals that are still pending. 
+Then, as part of this proposal information you can find a proposal's id `id`.
+
+Using this, you can now vote on a proposal using the following neuron command.
+```
+type RegisterVote = record { 
+  vote : int32; 
+  proposal : opt NeuronId 
+};
+```
+The `vote` represents if the neuron should adopt or reject the proposal and the `proposal` is the proposal ID of the proposal that the neuron votes on.
+
+**Set a neuron's following**
+
+In case that a neuron's owner lacks the time or expertise to vote on some proposals, a neuron can be configured to vote automatically by following other neurons on a topic-by-topic basis.
+For any valid topic, a list of followees can be specified, and the neuron will follow the vote of a majority of the followers on a proposal with a type belonging to that topic. If a null topic is specified, this acts as a catch-all that enables the neuron to follow the vote of followees for all topics where no explicit rule has been specified. 
+There are two topics that are excluded from the catch-all: `Governance` and `SNS & Neuron's Fund`. 
+
+:::info
+It is important to understand that a neuron's vote is only cast if the majority of the follwed neurons agree on a decision. Find [here](https://internetcomputer.org/docs/current/developer-docs/daos/nns/nns-app-quickstart#neuron-following) more information and examples regarding the following-rules.
+
+:::
+
+To define a follow rule, use the following neuron command
+ ```
+type Follow = record {
+  topic : int32; 
+  followees : vec NeuronId 
+};
+```
+The argument `topic` defines for which proposal topic this following rule should be applied.
+If the topic is `null`, then this rule is applied for the catch-all and will be applied for all topics where no other explicit rule is defined except for`Governance` and `SNS & Neuron's Fund`.
+The second argument `followees` defines a list of neuron IDs whose decision will be followed. There can be up to 15 followees for each topic (but more followees is not necessarily better - see the rules linked above).
```

Comment:
TODO: HOW TO GET THE ID OF TOPICS?

Reply:
https://github.com/dfinity/ic/blob/7184d168d44f3ba27f750285a0fba43a0c56190a/rs/nns/governance/src/gen/ic_nns_governance.pb.v1.rs#L3142 The IDs are specified here

Reply:
Thanks. I now added this, would be great if you could double check whether the explanation makes sense!

Reply:
For some reason I don't see where you added it 🤔 Could you point out where? (and maybe DM me so I can respond immediately)